diff -Naur python-telegram-bot-8.0.0/telegram/utils/request.py python-telegram-bot-8.0.0_b/telegram/utils/request.py
--- python-telegram-bot-8.0.0/telegram/utils/request.py	2017-09-01 10:38:15.000000000 +0200
+++ python-telegram-bot-8.0.0_b/telegram/utils/request.py	2017-09-17 15:05:39.690363214 +0200
@@ -30,10 +30,10 @@
 
 import certifi
 try:
-    import telegram.vendor.ptb_urllib3.urllib3 as urllib3
-    import telegram.vendor.ptb_urllib3.urllib3.contrib.appengine as appengine
-    from telegram.vendor.ptb_urllib3.urllib3.connection import HTTPConnection
-    from telegram.vendor.ptb_urllib3.urllib3.util.timeout import Timeout
+    import urllib3 as urllib3
+    import urllib3.contrib.appengine as appengine
+    from urllib3.connection import HTTPConnection
+    from urllib3.util.timeout import Timeout
 except ImportError:  # pragma: no cover
     warnings.warn("python-telegram-bot wasn't properly installed. Please refer to README.rst on "
                   "how to properly install.")
@@ -91,7 +91,7 @@
         kwargs = dict(
             maxsize=con_pool_size,
             cert_reqs='CERT_REQUIRED',
-            ca_certs=certifi.where(),
+            ca_certs="/etc/ssl/certs/ca-certificates.crt",
             socket_options=sockopts,
             timeout=urllib3.Timeout(
                 connect=self._connect_timeout, read=read_timeout, total=None))
